packages/core/src/RenderingEngine/VolumeViewport.ts

```diff
@@ -79,6 +83,21 @@ class VolumeViewport extends Viewport implements IVolumeViewport {
     return false;
   }
 
+  private initializeVolumeNewImageEventDispatcher(): void {
+    this.element.addEventListener(
```

Comment:
You need to also remove this event handler when the element is disabled. Otherwise if you re-enable it as, say, a stack viewport, we will still be reacting to CAMERA_MODIFIED events and this will likely break or just fire erroneous misleading events.